MVVMDemoNew/Cart/CartViewModel.cs

```diff
@@ -38,7 +38,7 @@ public CartViewModel()
         {
             CartItems = new ObservableCollection<ShoppingItem>()
             {
-                new ShoppingItem(new Guid(), "Navy Pants", "Navy is an terrible color for leg covers.", "pack://application:,,,/MVVMDemoNew;component/Images/navy pants.jpg", 1),
+                new ShoppingItem(new Guid(), "Navy Pants", "Navy is a horrible color for pants.", "pack://application:,,,/MVVMDemoNew;component/Images/navy pants.jpg", 1),
```

Comment:
Navy is a great color!

Reply:
I think Navy is the worst. We should leave it as is.